OrbitLinuxTracing/LinuxTracingUtils.cpp

```diff
@@ -72,15 +69,21 @@ std::vector<pid_t> GetAllPids() {
 }
 
 std::vector<pid_t> GetTidsOfProcess(pid_t pid) {
-  fs::directory_iterator proc_pid_task{fs::path{"/proc"} / std::to_string(pid) / "task"};
-  std::vector<pid_t> tids;
+  fs::directory_iterator proc_pid_task;
+  try {
+    proc_pid_task = fs::directory_iterator{fs::path{"/proc"} / std::to_string(pid) / "task"};
```

Comment:
maybe use exception-free variant instead?

Reply:
I was about to write the same. `directory_iterator::directory_iterator` has an overloard which takes a `std::error_code&` which can be checked, instead of throwing an exception. Ref: https://en.cppreference.com/w/cpp/filesystem/directory_iterator/directory_iterator

Reply:
Ah better, I had missed that, please have a look.